assets/css/main.css

```diff
@@ -55,18 +55,15 @@ article {
   color: var(--ins-color);
 }
 .error,
-a:not([type="button"]).destructive {
-  color: var(--del-color);
+.destructive {
+  --primary: var(--del-color);
+  --dropdown-color: var(--del-color);
+  --primary-hover: var(--del-color);
```

Comment:
Make the destructive feature a bit more generic, so that we can use for example `class="outline"` on buttons